### ISSUE TITLE
Selecting my PC returns an empty list

### ISSUE THREAD

**Issue by filipeaguiar** [moonlight-stream/moonlight-android#229]
I was playing The Witcher III on my PC, then I decided to play it on my Moto X on the bedroom. I oppened the app and selected the game. It showed an error saying that the game was already running (it wasnt). So, to be sure, I've restarted my PC and oppened my app again.

Then when I select my PC, from the Moonlight app, it doesn't show *anything*. I didn't change any network config, I didn't change any GeForce Experience settings, I didn't change any settings on the app. The list just vanishes. It didn't even throw an error os some kind. Nothing.

I've tried rebooting the PC again. Nothing.
I've tried rebooting my phone. Nothing either.

**Comment by Jan200101** [comment]
I actually got the same... kinda
I bought a controller with a mount for my phone for this app and downloaded it again now after updating my drivers and restarting 4 times I figured that it was nothing with my pc rather with the app

tl;dr
There are no games showing up

**Comment by cgutman** [comment]
It's still a GeForce Experience problem not Moonlight. When the app list is requested, it returns no entries. Try uninstalling and reinstalling GeForce Experience.

**Comment by Jan200101** [comment]
@cgutman welp thats funny I had to do that 3 times this year already because it kept getting stuck with not beeing able to connect to NVIDIA

**Comment by Jan200101** [comment]
@cgutman New Error (for me) its kinda the same as before but now it only displays Steam and opening that will result in a connection error

**Comment by cgutman** [comment]
Can you try rescanning for games in GFE?

**Comment by filipeaguiar** [comment]
Rescanning didn't work for me. I had to re-install the GeForce Experience to make it work. But it DID work. Thank you.

**Comment by Jan200101** [comment]
I had to reinstall 3 times and set the stream service to restart if starting failed...
but now it works
